docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -315,6 +315,217 @@ If you were using them, you need to replace them with the following code:
     UseDateTimeFieldProps<TDate, TEnableAccessibleFieldDOMStructure>;
   ```
 
+- `BaseSingleInputFieldProps`
+
+  - If you are building a custom field for a Date Picker:
+
+    ```diff
+    -import {
```

Comment:
The old code is always with the old generic (`TDate` and `TValue`), not the new version of `BaseSingleInputFieldProps` that people can't use anymore anyway).